graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java

```diff
@@ -91,6 +99,52 @@ protected HttpResponse doPost(ServiceRequestContext ctx, HttpRequest request) th
             return unsupportedMediaType();
         }
 
+        // See https://github.com/jaydenseric/graphql-multipart-request-spec/blob/master/readme.md
+        if (contentType.is(MediaType.MULTIPART_FORM_DATA)) {
+            return HttpResponse.from(FileAggregatedMultipart.aggregateMultipart(ctx, request)
+                                                            .thenApply(multipart -> {
+                try {
+                    final String operationsParam = toStringFromMultipartParam("operations", multipart.params());
+                    final String mapParam = toStringFromMultipartParam("map", multipart.params());
```

Comment:
`toString` seems not to make sense. Because its value is already `String`.
```suggestion
                    final ListMultimap<String, String> multipartParams = multipart.params();
                    final String operationsParam = getValueFromMultipartParam("operations", multipartParams);
                    final String mapParam = getValueFromMultipartParam("map", multipartParams);
```